Multisig/Features/Security/KeychainItem.swift

```diff
@@ -110,6 +110,9 @@ enum KeychainItem {
             ]
             if let context = LAContext(password: password) {
                 privateKeyAttrs[kSecUseAuthenticationContext] = context
+            } else {
+                // In case password is worng
```

Comment:
I would also add short explanation here regarding case when device password is not set (failure to create LAContext)